src/Microsoft.FeatureManagement/IFeatureDefinitionProvider.cs

```diff
@@ -13,18 +13,33 @@ namespace Microsoft.FeatureManagement
     public interface IFeatureDefinitionProvider
```

Comment:
Should we split it into separate interfaces for feature flags and dynamic features so users don't have to implement both if they only care about one but not the other?

Reply:
We split IFeatureManager and IDynamicFeatureManager so I'm not opposed to carrying that over here.